### DISCUSSION TITLE
Better way of informing users about qualifying to be on the leaderboard

### DISCUSSION THREAD

**Discussion by mikelerch** [monkeytypegame/monkeytype#5307] [Ideas]
I recently started using monkeytype and kept seeing the following message when I viewed the all-time leaderboard
> Your account must have 2 hours typed to be placed on the leaderboard.

After having typed for 2h, I was very excited to see my all-time ranking, so I went ahead to check but a new message showed up at the bottom, stating
> Not qualified

My initial thought was that my old records don't count, but it turns out I just had to wait until the leaderboard updated.

Maybe this message could be adjusted to give a bit more information to prevent confusion.

**Comment by fehmer** [comment]
all-time leaderboard is updated every 15 minutes. If you improve your PB it takes up to 15 to reflect as well. There is already a timer on top:
![image](https://github.com/monkeytypegame/monkeytype/assets/3728838/95c75f65-8c27-41d0-812e-5424612e060d)

**Reply by mikelerch** [reply]
Yes, I understand. However the message "Not qualified" doesn't really state much. If you just look at the leaderboard, you don't know that you have to wait for the next update in 15 minutes.

It's a minor issue, just wanted to bring it up because it wasn't clear to me at first.
